apps/etterlatte-behandling/src/main/kotlin/Contekst.kt

```diff
@@ -3,19 +3,26 @@ package no.nav.etterlatte
 import kotlinx.coroutines.runBlocking
 import no.nav.etterlatte.behandling.EnhetService
 import no.nav.etterlatte.common.Enheter
-import no.nav.etterlatte.config.AzureGroup
 import no.nav.etterlatte.libs.ktor.AZURE_ISSUER
+import no.nav.etterlatte.tilgangsstyring.SaksbehandlerMedRoller
+import no.nav.etterlatte.token.BrukerTokenInfo
+import no.nav.etterlatte.token.Saksbehandler
+import no.nav.etterlatte.token.Systembruker
 import no.nav.security.token.support.core.context.TokenValidationContext
 import no.nav.security.token.support.v2.TokenValidationContextPrincipal
-import org.slf4j.LoggerFactory
+import tilgangsstyring.AzureGroup
 import java.sql.Connection
 
 object Kontekst : ThreadLocal<Context>()
 
 class Context(
     val AppUser: User,
     val databasecontxt: DatabaseKontekst
-)
+) {
+    fun appUserAsSaksbehandler(): SaksbehandlerMedEnheterOgRoller {
+        return this.AppUser as SaksbehandlerMedEnheterOgRoller
```

Comment:
Åpner vi her for at man antar at det er saksbehandler når det kanskje ikke er det?

Reply:
Absolutt